emulate-browsers/base/injected-scripts/polyfill.additions.ts

```diff
@@ -1,7 +1,7 @@
 for (const addition of args.additions || []) {
   try {
     if (addition.propertyName === 'getVideoPlaybackQuality') {
-      addition.property['_value()'] = function() {
+      addition.property['_$$value()'] = function() {
```

Comment:
Double $ on purpose? I'm actually fine with it, just curious

Reply:
Yes, was trying to separate the custom meta keys used in the DA export from the custom keys being added in SA.

Reply:
It was more just for my sanity. I'm not against dropping it to a single $ if you think it best.